libs/cdk/utils/services/keyboard-support/keyboard-support.service.ts

```diff
@@ -45,7 +47,7 @@ export class KeyboardSupportService<T> {
         this._tabKeyNavigation = tabKeyNavigation;
         this._keyManager = new FocusKeyManager(queryList).withWrap(wrap).withHomeAndEnd();
```

Comment:
If this function can be called multiple times, should not we destroy the old _keyManager first before creating new one? like we do in destroy hook?